Formula/vc.rb

```diff
@@ -19,10 +19,13 @@ class Vc < Formula
   depends_on "cmake" => :build
 
   def install
-    mkdir "build" do
-      system "cmake", "..", "-DBUILD_TESTING=OFF", *std_cmake_args
-      system "make", "install"
-    end
+    # not filter out -march
+    # see discussion in here, https://github.com/VcDevel/Vc/issues/348#issuecomment-1328101410
+    ENV.runtime_cpu_detection
```

Comment:
Based on audit, this is probably not right way and may end up optimizing binaries more than we want. If so, will need to use CMake arguments to override host CPU detection.

Reply:
Any code suggestion?